client/src/styles/MilestoneTable.css

```diff
@@ -0,0 +1,25 @@
+.table-wrapper {
+    display: flex;
+    width: 100%;
+    height: 100px;
+    justify-content: center;
+    align-items: center;
+    border-radius: 7px;
+    background-color: #f5f5f5;
+/* #eef4f9 */
+}
+table{
+    width: 88%;
+    border-collapse: collapse;
+}
+th {
+    border: 1px solid #808080;
+    background-color: #f2f2f2;
+    padding: 5px;
+}
+td {
+    border: 1px solid #808080;
+    text-align: center;
+    background-color: #ffffff;
+    padding: 10px;
+}
```

Comment:
good job as the other one we need to think about some css root and default for now its great